<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the public 'HWI\Bundle\OAuthBundle\Controller\LoginController' shared service.

$this->services['HWI\\Bundle\\OAuthBundle\\Controller\\LoginController'] = $instance = new \HWI\Bundle\OAuthBundle\Controller\LoginController(($this->services['security.authentication_utils'] ?? $this->load('getSecurity_AuthenticationUtilsService.php')), ($this->services['router'] ?? $this->getRouterService()), ($this->services['security.authorization_checker'] ?? $this->getSecurity_AuthorizationCheckerService()), ($this->services['session'] ?? $this->load('getSessionService.php')), false, 'IS_AUTHENTICATED_REMEMBERED');

$instance->setContainer(($this->privates['.service_locator.E18xQID'] ?? $this->load('get_ServiceLocator_E18xQIDService.php'))->withContext('HWI\\Bundle\\OAuthBundle\\Controller\\LoginController', $this));

return $instance;
